.github/workflows/jepsen.yml

```diff
@@ -0,0 +1,65 @@
+# Runs a Jepsen test - cas-register (no nemesis) by default.
+#   See inputs for various options.
+#   Repo: https://github.com/tendermint/jepsen
+#
+# If you want to test a new breaking version of Tendermint, you'll need to
+# update the Merkleeyes ABCI app and 'merkleeyesUrl' input accordingly. You can
+# upload a new tarball to
+# https://github.com/tendermint/jepsen/releases/tag/0.2.1.
+#
+# Manually triggered.
+name: jepsen
```

Comment:
Is there any plan to run and maintain these tests? I think it's fine to keep them, but I don't think anyone's run them in years *and* I don't know that anyone ever ran them on 0.34.x, so they may not work at all. Worth checking before we commit it, I think.

Reply:
No plans here, I can remove them for now.